api_tests/nodes/views/test_node_forks_list.py

```diff
@@ -318,3 +322,18 @@ def test_read_only_contributor_can_fork_private_registration(self, app, private_
         assert res.status_code == 201
         assert res.json['data']['id'] == private_project.forks.first()._id
         assert res.json['data']['attributes']['title'] == 'Fork of ' + private_project.title
+
+    def test_send_email_success(self, app, user, public_project_url, fork_data_with_title, public_project):
+        res = app.post_json_api(public_project_url, fork_data_with_title, auth=user.auth)
+        assert res.status_code == 201
+        assert res.json['data']['id'] == public_project.forks.first()._id
+        assert mails.send_mail.called_with(user.email, mails.FORK_COMPLETED, title= res.json['data']['attributes']['title'], guid=res.json['data']['id'], mimetype='html')
+
+    def test_send_email_failed(self, app, user, public_project_url, fork_data_with_title, public_project):
```

Comment:
Can you confirm that this test failed before you fixed the mail? It checks if the mail function is called but since it is mocked it isn't clear that it would catch issues.